### ISSUE TITLE
Coupon usage limit has been reached

### ISSUE THREAD

**Issue by saltsurfsea** [woocommerce/woocommerce#23846]
We don't offer technical support on GitHub so we recommend using the following:

**Reading our documentation**
Usage docs can be found here: https://docs.woocommerce.com/

If you have a problem, you may want to start with the self help guide here: https://docs.woocommerce.com/document/woocommerce-self-service-guide/

**Technical support for premium extensions or if you're a WooCommerce.com customer**
 from a human being - submit a ticket via the helpdesk
https://woocommerce.com/contact-us/ 

**General usage and development questions**
- WooCommerce Slack Community: https://woocommerce.com/community-slack/
- WordPress.org Forums: https://wordpress.org/support/plugin/woocommerce
- The WooCommerce Help and Share Facebook group

**Comment by saltsurfsea** [comment]
Hi there,

When I give out a free product/s I create and process an order backend so the items reduce off my inventory.
I use a coupon code for 100% off the total order, with no expiry date or any restrictions.  This has been working fine until I recently updated to 3.6.4 and as a guest user I get the message  "coupon usage limit has been reached". I created a new coupon code again for 100% off and no expiry date and the same message "coupon usage limit has been reached" again came up.  When I processed the order using my email address as a customer it worked.

I also tried this from the website (not backend) and it said "Coupon code applied successfully" but did not give the 100% discount.

I have tried deactivating all plugins and trying the coupon code and again the same error message came up.

How can this be fixed as I have customers who might process an order as a guest user and if they use a coupon code it may create issues.

Thanks
